tx-pool/src/component/pending.rs

```diff
@@ -70,6 +70,16 @@ impl PendingQueue {
             }
         }
     }
+
+    pub(crate) fn drain(&mut self) -> Vec<TransactionView> {
+        let txs = self
+            .inner
+            .values()
+            .map(|entry| entry.transaction().clone())
+            .collect::<Vec<_>>();
+        self.inner.clear();
```

Comment:
`::std::mem::take` maybe better?

Reply:
I think this is trivial.

So I decide to ignore it.

After all high priority tasks, I will do this replacement for whole workspace.